modules/tensor_mechanics/src/materials/ComputeThermalExpansionEigenStrain.C

```diff
@@ -12,9 +12,10 @@ InputParameters validParams<ComputeThermalExpansionEigenStrain>()
 {
   InputParameters params = validParams<ComputeStressFreeStrainBase>();
   params.addClassDescription("Computes Eigenstrain due to thermal expansion");
-  params.addCoupledVar("temperature", 273, "Coupled temperature in units of Kelvin");
+  params.addCoupledVar("temperature", "Coupled temperature in K");
```

Comment:
It shouldn't have the "in K" part. By removing the 273 default, it is now completely unit system independent.